lib/float/common.sail

```diff
@@ -86,4 +86,13 @@ function float_decompose(op) = {
   }
 }
 
+val      not : bool -> bool
```

Comment:
There is already a `not_bool` function in flow.sail: https://github.com/rems-project/sail/blob/sail2/lib/flow.sail

sail-riscv has the following to add `not()`:

```
// not_bool alias.
val not : forall ('p : Bool). bool('p) -> bool(not('p))
function not(b) = not_bool(b)
```

Reply:
Great! Looks we can alias this similar to sail-riscv, let me update it.